app/api/server/v1/roles.js

```diff
@@ -13,6 +13,19 @@ API.v1.addRoute('roles.list', { authRequired: true }, {
 	},
 });
 
+// api to to get updated roles after a date(in ISODate format)
+API.v1.addRoute('roles.listByUpdatedDate', { authRequired: true }, {
+	get() {
+		check(this.bodyParams, {
+			updatedAfter: String,
```

Comment:
@ashwaniYDV can you please validate this query parameter, to make sure it is a valid date? Here we have an example: https://github.com/RocketChat/Rocket.Chat/blob/develop/app/api/server/v1/rooms.js#L34-L43

Reply:
Done :+1: